packages/jest-cli/src/cli/args.js

```diff
@@ -91,6 +91,11 @@ const options = {
       'rare.',
     type: 'boolean',
   },
+  testNamePattern: {
```

Comment:
Maybe `t`? I would reserve `p` for filtering by path, because it means that in the watch mode.